static/app/components/deprecatedforms/input.tsx

```diff
@@ -1,13 +1,8 @@
 import classNames from 'classnames';
-import omit from 'lodash/omit';
 
 interface Props extends React.InputHTMLAttributes<HTMLInputElement> {}
 
 export default function Input({className, ...otherProps}: Props) {
-  return (
-    <input
-      className={classNames('form-control', className)}
-      {...omit(otherProps, 'children')}
-    />
-  );
+  const {children: _, ...props} = otherProps;
```

Comment:
Children keyword can be omitted by the typescript definition of Props of this component. It'll make the code easier.

Reply:
Yes, I'm just unclear if there is a place where we do it. I'll try to make the change and see if TS complains about any occurrences. If it doesn't I'll replace the rest of the implementations